pmm-app/src/react-plugins-deps/components/Table/Table.tsx

```diff
@@ -0,0 +1,115 @@
+// @ts-nocheck
```

Comment:
Yes, sorry about that, I did it because need to spend some time to investigate how to set type of data returned from useTable hook, it's pretty complex, removed ts-nocheck and replace it with ts-ignore where it needed